tests/lax_numpy_test.py

```diff
@@ -1565,6 +1565,20 @@ def testDiagIndices(self, ndim, n):
     np.testing.assert_equal(np.diag_indices(n, ndim),
                              jnp.diag_indices(n, ndim))
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "arr_shape={}".format(
+        jtu.format_shape_dtype_string(shape, dtype)
+      ),
+       "dtype": dtype, "shape": shape}
+      for dtype in default_dtypes
+      for shape in [(1,1), (2,2), (3,3), (4,4), (5,5)]))
+  def testDiagIndicesFrom(self, dtype, shape):
+    rng = jtu.rand_default(self.rng())
+    np_fun = lambda arg : np.diag_indices_from(arg)
+    jnp_fun = lambda arg : jnp.diag_indices_from(arg)
```

Comment:
The ``lambda`` wrapper is superfluous here: you can use ``np_fun = np.diag_indices_from`` and similar for ``jnp``.